index.html

```diff
@@ -0,0 +1,157 @@
+          <li>Javascript</li>
+        </ul>
+        <div class="but"><button class="button">See project</button></div>
+      </div>
+
+      <div class="card">
+        <img src="./images/2.png" alt="">
+        <p class="text-title">Multi-post<br>stories</p>
+        <div class="experience">
+        <span class="subtitle">CANOPY</span>
+        <ul class="block">
+          <li>&bull; Back End Dev</li>
+          <li>&bull; 2015</li>
+        </ul>
+        </div>
+        <p class="text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul class="skills">
+          <li>html</li>
+          <li>css</li>
+          <li>Javascript</li>
+        </ul>
+        <div class="but"><button class="button">See project</button></div>
+      </div>
+
+      <div class="card">
+        <img src="./images/3.png" alt="">
+        <p class="text-title">Tonic</p>
+        <div class="experience">
+        <span class="subtitle">CANOPY</span>
+        <ul class="block">
+          <li>&bull; Back End Dev</li>
+          <li>&bull; 2015</li>
+        </ul>
+        </div>
+        <p class="text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul class="skills">
+          <li>html</li>
+          <li>css</li>
+          <li>Javascript</li>
+        </ul>
+        <div class="but"><button class="button">See project</button></div>
+      </div>
+
+      <div class="card">
+        <img src="./images/4.png" alt="">
+        <p class="text-title">Multi-post<br>stories</p>
+        <div class="experience">
+        <span class="subtitle">CANOPY</span>
+        <ul class="block">
+          <li>&bull; Back End Dev</li>
+          <li>&bull; 2015</li>
+        </ul>
+        </div>
+        <p class="text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul class="skills">
+          <li>html</li>
+          <li>css</li>
+          <li>Javascript</li>
+        </ul>
+        <div class="but"><button class="button">See project</button></div>
+      </div>
+      
+
+      <section class="about-myself">
+        <div class="introduction">
+          <h1>About<br>Myself</h1>
+          <p class="about">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p><br>
+          <a href="#" class="connect">LET'S CONNECT</a><br><br>
+           <div class="socials">
+           <ul>
+            <li><a href=""><img src="./images/github.png" alt=""></a></li>
+            <li><a href=""><img src="./images/linkedin.png" alt=""></a></li>
+            <li><a href=""><img src="./images/v.png" alt=""></a></li>
+            <li><a href=""><img src="./images/twitter.png" alt=""></a></li>
+            <li><a href=""><img src="./images/m.png" alt=""></a></li>
```

Comment:
[OPTIONAL] Kindly give the `alt` attr a descriptive text because this will increase your website's accessibility by screen readers a clear idea of what the image is about (maybe something like a twitter link...etc)